conjur/init/init_controller.py

```diff
@@ -0,0 +1,137 @@
+        # pylint: disable=logging-fstring-interpolation
+        logging.debug(f"Initiating a TLS connection with '{self.conjurrc_data.appliance_url}'")
+        fingerprint, fetched_certificate = self.init_logic.get_certificate(url.hostname, url.port)
+
+        sys.stdout.write(f"\nThe server's certificate SHA-1 fingerprint is:\n{fingerprint}\n")
+        sys.stdout.write("\nTo verify this certificate, it is recommended to run the following " \
+                         "command on the Conjur server:\n" \
+                         "openssl x509 -fingerprint -noout -in ~conjur/etc/ssl/conjur.pem\n\n")
+        trust_certificate = input("Trust this certificate? [no]: ")
+        if trust_certificate.lower() != 'yes':
+            raise ValueError("You decided not to trust the certificate")
+
+        return fetched_certificate
+
+    def get_account_info(self, conjurrc_data):
+        """
+        Method to fetch the account from the user
+        """
+        if conjurrc_data.account is None:
+            try:
+                self.init_logic.fetch_account_from_server(self.conjurrc_data)
+            # pylint: disable=broad-except,logging-fstring-interpolation
+            except Exception as error:
+                # pylint: disable=line-too-long,logging-fstring-interpolation
+                logging.debug(f"Unable to fetch the account from the Conjur server. Reason: {error}")
+                # If there was a problem fetching the account from the server, we will request one
+                conjurrc_data.account = input("Enter your organization account name: ")
+
+                if conjurrc_data.account is None or conjurrc_data.account == '':
+                    # pylint: disable=raise-missing-from
+                    raise RuntimeError("Error: account is required")
+
+    def write_certificate(self, fetched_certificate):
+        """
+        Method to write the certificate fetched from the Conjur endpoint on the user's machine
+        """
+        url = urlparse(self.conjurrc_data.appliance_url)
+        if self.conjurrc_data.cert_file is None and url.scheme == "https":
+            # pylint: disable=line-too-long
+            self.conjurrc_data.cert_file = os.path.join(os.path.dirname(conjur.constants.DEFAULT_CONFIG_FILE),
+                                            conjur.constants.CERTIFICATE_FILENAME)
+            is_file_written = self.init_logic.write_certificate_to_file(fetched_certificate,
+                                                                       self.conjurrc_data.cert_file,
+                                                                       self.force_overwrite)
+            if not is_file_written:
+                self.__ensure_overwrite_file(self.conjurrc_data.cert_file)
+                self.init_logic.write_certificate_to_file(fetched_certificate,
+                                                          self.conjurrc_data.cert_file,
+                                                          True)
```

Comment:
Add explicit argument name:
```suggestion
                                                          force_overwrite_flag=True)
```